usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -35,29 +35,56 @@ The currently available endpoints are listed in the following table.
+|[/api/v2/download/awards](/api/v2/download/awards)|POST| Generates zip file for download of award data in CSV format|
+|[/api/v2/download/count](/api/v2/download/count)|POST| Returns the number of transactions that would be included in a download request for the given filter set|
+|[/api/v2/download/status](/api/v2/download/status)|GET| gets the current status of a download job that that has been requested with the v2/download/awards/ or v2/download/transaction/ endpoint that same day|
+|[/api/v2/download/transactions](/api/v2/download/transactions)|POST|Generates zip file for download of award data in CSV format|
+|[/api/v2/federal_accounts/](/api/v2/federal_accounts/)|POST| Returns financial spending data by object class``|
+|[/api/v2/federal_accounts/<ACCOUNT_CODE\>/](/api/v2/federal_accounts/020-0550/)|GET| Returns a federal account based on its federal account code|
+|[/api/v2/federal_accounts/<ACCOUNT_CODE\>/available_object_classes](/api/v2/federal_accounts/4324/available_object_classes)|GET| Returns financial spending data by object class based on account's internal ID|
+|[/api/v2/federal_accounts/<ACCOUNT_CODE\>/fiscal_year_snapshot](/api/v2/federal_accounts/4324/fiscal_year_snapshot)|GET| Returns budget information for a federal account for the most recent year based on account's internal ID|
+|[/api/v2/federal_accounts/<ACCOUNT_CODE\>/fiscal_year_snapshot/<YEAR\>](/api/v2/federal_accounts/4324/fiscal_year_snapshot/2017)|GET| Returns budget information for a federal account for the year provided based on account's internal ID|
+|[/api/v2/federal_obligations](/api/v2/federal_obligations/?fiscal_year=2019&funding_agency_id=315&limit=10&page=1)|GET| Returns a paginated list of obligations for the provided agency for the provided year|
+|[/api/v2/financial_balances/agencies/](/api/v2/financial_balances/agencies?fiscal_year=2017&funding_agency_id=4324)|GET| Returns financial balances by agency and the latest quarter for the given fiscal year|
+|[/api/v2/financial_spending/major_object_class/](/api/v2/financial_spending/major_object_class?fiscal_year=2017&funding_agency_id=4324)|GET| Returns financial spending data by object class for the latest quarter based on the given fiscal year |
+|[/api/v2/financial_spending/object_class/](/api/v2/financial_spending/object_class?fiscal_year=2017&funding_agency_id=4324&major_object_class_code=20)|GET| Returns financial spending data by object class for the latest quarter based on the given fiscal year|
+|[/api/v2/recipient/children/<DUNS_NUMBER\>/](/api/v2/recipient/children/006928857/)|GET| Returns recipient details based on DUNS number|
+|[/api/v2/recipient/duns/](/api/v2/recipient/duns/)|POST| Returns a list of recipients in USAspending DB|
+|[/api/v2/recipient/duns/<HASH_VALUE\>/](/api/v2/recipient/duns/HASH/)|GET| Returns a high-level overview of a specific recipient, given its id|
```

Comment:
I know the recipient hash is potentially ephemeral, however let's try using a large recipient from SAM:  CALIFORNIA, STATE OF: `42c19cbe-ced7-5d41-2f80-cd27a22b1575-P`